sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/FileFormat.scala

```diff
@@ -317,7 +317,8 @@ object FileFormat {
       fieldNames: Seq[String],
       filePath: SparkPath,
       fileSize: Long,
-      fileModificationTime: Long): InternalRow = {
+      fileModificationTime: Long,
+      fileStatus: FileStatus): InternalRow = {
```

Comment:
Here, `fileStatus` has been added as a parameter. So, do we still need to pass in `fileModificationTime`, `fileSize`, and `filePath` separately? We can obtain all of them through `fileStatus`, right?

Reply:
filePath, fileSize, and fileModificationTime are removed